### PR TITLE
packages.yml: add tmt with copr repo

### PR DESCRIPTION
**Pull request by optak**
This change is needed to run tmt tests via libvirt.

### CODE REVIEW

**Comment by pvalena** [comment]
Great!

Does this mean the PRs tests' will run on top of copr-built package?

**Comment by pvalena** [comment]
_Note: I have a prototype of an `artifact installator` in the works._

File: plans/packages.yml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 - hosts: all
   tasks:
+  - name: Install tmt rpeository
```
Comment (psss) [inline plans/packages.yml]: A small typo in `repository`.

File: plans/packages.yml [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+    yum_repository:
+      name: tmt
+      description: tmt repo
+      baseurl: https://copr-be.cloud.fedoraproject.org/results/psss/tmt/fedora-$releasever-$basearch/
```
Comment (psss) [inline plans/packages.yml]: So this will work for Fedora only, right? We can start with that and add epel later.

Comment (pvalena) [reply plans/packages.yml]: I'm still thinking: if you're running tests, don't you wan't to build and install the _local_  modifications (can we copy git from local dir path?). Is there some sane way to do so? (I know of `git archive ...` which would be nice to use here IMO.)

@psss WDYT?

Comment (psss) [reply plans/packages.yml]: Good question. So I expect the common workflow will be something like what we have outlined in the [smoke](https://tmt.readthedocs.io/en/devel/stories/cli.html#smoke) story:

    > tmt story show smoke
    /stories/cli/run/smoke
           story As a developer I want to do a quick smoke test of my
                 freshly built rpm package.
         example tmt run --all \
                 prepare --artifact=tmt-0.4-1.fc29.noarch.rpm \
                 execute --script='tmt --help'

In this way it should be possible to easily test the fresh package. We could possibly make that even more user friendly in the future but for the start I think this should be good enough.

Comment (optak) [reply plans/packages.yml]: I don't get it, @psss . Why should it be a problem to have repo for fedora only? There is already fmf as rpm, which seems pretty fedora focused part of playbook. fmf package is not available in RHEL (afaik) and ansible module for dnf probably don't work with other packaging systems either.
If I should rewrite this MR, I will need to understand your idea better.

Comment (psss) [reply plans/packages.yml]: @optak, sorry for slow response. The thing is that test should work fine with epel too so the setup should not be fedora only. Anyway, `tmt` package is now [available](https://bodhi.fedoraproject.org/updates/?packages=tmt) in Fedora, so for quick smoke testing we are set. For testing freshly built rpms we still need implement the `--artifact` option.

**Comment by psss** [comment]
Adding a copr repository is no more necessary. Stable `tmt` package available directly in Fedora.

**Pull request closed without merging**